pandas/core/groupby/groupby.py

```diff
@@ -1484,6 +1484,7 @@ def _numba_agg_general(
         func: Callable,
         dtype_mapping: dict[np.dtype, Any],
         engine_kwargs: dict[str, bool] | None,
+        is_grouped_kernel: bool = False,
```

Comment:
Technically can remove this now, since all the existing numba kernels have been ported to have a groupby variant.

If you'd rather have me split this PR by kernel, this would be necessary (would remove after all the PRs land).